src/Widgets/DataIconWidget/DataIconWidgetComponent.tsx

```diff
@@ -33,40 +34,43 @@ provideComponent(DataIconWidget, ({ widget }) => {
   )
 })
 
-const AttributeValue = connect(function AttributeValue({
-  conditions,
-  fallbackIcon,
-  size,
-}: {
-  conditions: DataIconConditionWidgetInstance[]
-  fallbackIcon: string
-  size: string
-}) {
-  const attributeValue = ensureString(useData().dataItemAttribute()?.get())
-  const matchingCondition =
-    attributeValue &&
-    conditions.find(
-      (condition) => condition.get('attributeValue') === attributeValue,
-    )
+const AttributeValue = connect(
+  function AttributeValue({
+    conditions,
+    fallbackIcon,
+    size,
+  }: {
+    conditions: DataIconConditionWidgetInstance[]
+    fallbackIcon: string
+    size: string
+  }) {
+    const attributeValue = ensureString(useData().dataItemAttribute()?.get())
+    const matchingCondition =
+      attributeValue &&
+      conditions.find(
+        (condition) => condition.get('attributeValue') === attributeValue,
+      )
 
-  const matchingOption = useEnumOptions().find(
-    ({ value }) => value === attributeValue,
-  )
-  const title = matchingOption?.title ?? attributeValue
+    const matchingOption = useEnumOptions().find(
+      ({ value }) => value === attributeValue,
+    )
+    const title = matchingOption?.title ?? attributeValue
 
-  return (
-    <IconComponent
-      icon={
-        matchingCondition
-          ? matchingCondition.get('icon') || 'bi-box'
-          : fallbackIcon
-      }
-      size={size}
-      link={null}
-      title={title || localizeNotAvailable()}
-    />
-  )
-})
+    return (
+      <IconComponent
+        icon={
+          matchingCondition
+            ? matchingCondition.get('icon') || 'bi-box'
+            : fallbackIcon
+        }
+        size={size}
+        link={null}
+        title={title || localizeNotAvailable()}
+      />
+    )
+  },
+  { loading: Loading },
```

Comment:
* [ ] FYI since the icon is basically an image we could go with a more decent placeholder like the generic browser default for images that are still loading. E.g. `{ loading: () => null }`